### ISSUE TITLE
Table markup doubles user name with alt text

### ISSUE THREAD

**Issue by AnandChowdhary** [all-contributors/all-contributors#323]
**Describe the bug**
The HTML markup of a typical table cell generated by All Contributors looks like this:

```html
<td align="center">
    <a href="https://anandchowdhary.com/?utm_source=github&utm_campaign=about-link"><img src="https://avatars3.githubusercontent.com/u/2841780?v=4" width="100px;" alt="Anand Chowdhary" />
        <br /><sub><b>Anand Chowdhary</b></sub></a>
    <br /><a href="https://github.com/staart/api/commits?author=AnandChowdhary" title="Code">💻</a> <a href="https://github.com/staart/api/commits?author=AnandChowdhary" title="Documentation">📖</a> <a href="#design-AnandChowdhary" title="Design">🎨</a></td>
```

As you'll notice, the `<img>` has alternate text with the user's name, which is then repeated below in the `<sub><b>` tag.

The problem is that a screen reader would read this is "Anand Chowdhary Anand Chowdhary", when it should actually read it as "Anand Chowdhary". The solution is for the image to be `<img alt="">`. Empty alternate text means a screen reader should not read this image out, and is perfectly valid markup.

**Comment by AnandChowdhary** [comment]
I think I should open this issue in the [all-contributors-bot](https://github.com/all-contributors/all-contributors-bot) project instead.

**Comment by Berkmann18** [comment]
So you think images should be ignored by screen readers?

Also, this is done by the CLI, not the bot (even if you use the bot).

**Comment by AnandChowdhary** [comment]
> So you think images should be ignored by screen readers?

In this case, yes, since the images are purely decorative and do not provide any additional information. The information they do provide is the name of the user, which is repeated right below it. Please read the [decorative images section of the WCAG](https://www.w3.org/WAI/tutorials/images/decorative/):

> Image with adjacent text alternative: This picture of a sleeping dog is already sufficiently described by the adjacent text. There is no need to repeat this information, and a null (empty) alt value can be used for this image.

```html
<p>
	<img src="sleepingdog.jpg" alt="">
	<strong>The sleeping dog:</strong> ...
</p>
```

I've opened the updated issue here: https://github.com/all-contributors/all-contributors-cli/issues/217

**Comment by Berkmann18** [comment]
> In this case, yes, since the images are purely decorative and do not provide any additional information. The information they do provide is the name of the user, which is repeated right below it. Please read the decorative images section of the WCAG:

Makes sense and I agree.
Feel free to open a PR on the CLI repo and I (or whoever can) will look into it when I can.
